mutation_partition_view.cc

```diff
@@ -259,12 +259,12 @@ future<> mutation_partition_view::do_accept_gently(const column_mapping& cm, Vis
     read_and_visit_row(mpv.static_row(), cm, column_kind::static_column, static_row_cell_visitor{visitor});
     co_await coroutine::maybe_yield();
 
-    for (auto&& rt : mpv.range_tombstones()) {
+    for (auto& rt : mpv.range_tombstones_range()) {
```

Comment:
I think we should just replace the eager variants with the new lazy variant everywhere. Code wanting to create a copy of the container can do so with the iterator constructor of the respective container.

Reply:
I'll take a stab at it in the next rev.